### PR TITLE
feature: add feature to increase/decrease size

### PR DESCRIPTION
**Pull request by be-ndee**

### CODE REVIEW

**Comment by gegoune** [comment]
I never used `NvimTreeResize` before but how about bundling new functionality into that command?

`NvimTreeResize 50` - as of now: resize to 50
`NvimTreeResize +10` - increment by 10
`NvimTreeResize -10` - decrement by 10

**Comment by be-ndee** [comment]
Good idea @gegoune ! I changed to code so you can use the NvimTreeResize function like you suggested it.

File: lua/nvim-tree/utils.lua [hunk 1]
```diff
@@ -245,4 +245,10 @@ function M.table_create_missing(tbl, sub)
   return t
 end
 
+-- @param value string
+-- @return string
+function M.trim_string(value)
+  return (string.gsub(value, "^%s*(.-)%s*$", "%1"))
+end
```
Comment (kyazdani42) [inline lua/nvim-tree/utils.lua:252]: uneeded, there is `vim.trim` function already :)

Comment (be-ndee) [reply lua/nvim-tree/utils.lua]: Ah, perfect. Searched for a similar function in native lua...

**Review by kyazdani42 — APPROVED** [review]

**Pull request merged**